compiler/codegen/FrontEnd.hpp

```diff
@@ -150,6 +150,7 @@ class TR_FrontEnd : public TR_Uncopyable
    virtual uint32_t offsetOfIsOverriddenBit();
 
    // Needs VMThread
+   virtual bool isMethodTracingEnabled(TR_OpaqueMethodBlock *method);
```

Comment:
I think a comment stating that `isMethodTracingEnabled()` is intended to replace the other methods and why that's the case would be appropriate here.